paddle/trainer/Trainer.cpp

```diff
@@ -88,21 +88,8 @@ P_DEFINE_string(model_list, "",
 
 namespace paddle {
 
-void Trainer::init(int argc, char** argv) {
-  initMain(argc, argv);
-  initPython(argc, argv);
-
-  auto config = TrainerConfigHelper::createFromFlagConfig();
-  feenableexcept(FE_INVALID | FE_DIVBYZERO | FE_OVERFLOW);
-
-  init(config);
-}
-
 void Trainer::init(const std::shared_ptr<TrainerConfigHelper> &config,
-                   bool testing,
-                   const std::shared_ptr<GradientMachine> &gradientMachine,
-                   const std::shared_ptr<DataProvider> &dataProvider,
-                   const std::shared_ptr<DataProvider> &testDataProvider) {
+                   bool testing) {
```

Comment:
This interface is used by other projects. Also used by recent pull request: https://github.com/baidu/Paddle/pull/193/files#diff-71eb11a1ef9cbe77f5e207dc915cc8b6